example/package.json

```diff
@@ -29,25 +29,25 @@
     ]
   },
   "dependencies": {
-    "@cumulus/api": "^1.11.2",
+    "@cumulus/api": "^1.11.3",
     "@cumulus/checksum": "^1.11.2-alpha3",
-    "@cumulus/cmrjs": "^1.11.2",
-    "@cumulus/common": "^1.11.2",
-    "@cumulus/deployment": "^1.11.2",
-    "@cumulus/discover-granules": "^1.11.2",
-    "@cumulus/discover-pdrs": "^1.11.2",
-    "@cumulus/hello-world": "^1.11.2",
-    "@cumulus/integration-tests": "^1.11.2",
-    "@cumulus/move-granules": "^1.11.2",
-    "@cumulus/parse-pdr": "^1.11.2",
-    "@cumulus/pdr-status-check": "^1.11.2",
-    "@cumulus/post-to-cmr": "^1.11.2",
-    "@cumulus/queue-granules": "^1.11.2",
-    "@cumulus/queue-pdrs": "^1.11.2",
-    "@cumulus/sf-sns-report": "^1.11.2",
-    "@cumulus/sync-granule": "^1.11.2",
-    "@cumulus/test-data": "^1.11.2",
-    "@cumulus/test-processing": "^1.11.2",
+    "@cumulus/cmrjs": "^1.11.3",
```

Comment:
It seems that the `"@cumulus/cmrjs"` and `"@cumulus/common"` packages are not updated to `1.11.3`

Reply:
Are we updating the version for all the packages?

Reply:
@jennyhliu The way lerna does it is it will publish a new version of all the packages to npm, but in the individual package.jsons it will only do an update if a file has changed. So for the files that did not have any change going from 1.11.2 to 1.11.3 it leaves them as is. Since I update example/package.json manually outside of lerna I update all of them.